website/client/src/components/static/appleRedirect.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div
+    class="static-view"
+    v-html="html"
+  ></div>
+</template>
+
+<style lang='scss'>
+@import '~@/assets/scss/static.scss';
+</style>
+
+<script>
+
+export default {
+  data () {
+    return {
+      html: '',
```

Comment:
I'm probably missing something but this property doesn't seem to be ever changed?

Reply:
You're right. I copied the page from a different static page and forgot to remove this